### ISSUE TITLE
Wrong parse when input a charater <

### ISSUE THREAD

**Issue by dnowba** [terrylinooo/githuber-md#14]
When write a code as below
```
<pre><code id="example">
for (let i = 0; i < 4; i++) 
</code></pre>
```
The character "<" will parse as a "open tag" but not "less than", that will mess up next content.
I know that the markdown use "```" to parse `<code>` element, but sometimes we need special format to express other language code.
Please help!

**Comment by terrylinooo** [comment]
is this issue  probably similar to #11 ?

**Comment by dnowba** [comment]
Yes, I think it is the same issue.

**Comment by terrylinooo** [comment]
@dnowba ok, I will make it in 1.6.1

**Comment by dnowba** [comment]
now the work aroud way is turn them to "Regular HTML character codes", for instance:
if you want show < , you have to type &lt;.
if you want show * , you have to type &#42;
I suggest that , you can Ignore content that contains in <pre> <code>, but not "shortcode" , because the shortcode developed in wordpress was often use to render as html, not preformat content.

**Comment by terrylinooo** [comment]
This will be fixed in 1.7.0, released soon.
